src/app/api/people/sync/route.ts

```
@@ -0,0 +1,305 @@
+          case 'nicknames':
+          case 'nickname':
+            // Ensure it's an array for multi_select
+            personData.nicknames = Array.isArray(value) ? value : (value ? [value] : []);
+            if (personData.nicknames && personData.nicknames.length > 0) {
+              console.log(`✓ Synced nicknames for ${personData.name}:`, personData.nicknames);
+            }
+            break;
+          default:
+            // Fallback: convert key to snake_case
+            const columnName = key.toLowerCase().replace(/[^a-z0-9_]/g, '_');
+            personData[columnName] = value;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)